README.md

```diff
@@ -288,6 +288,18 @@ nvm install 6 --reinstall-packages-from=5
 nvm install v4.2 --reinstall-packages-from=iojs
 ```
 
+Note that reinstalling packages _explicitly does not update the npm version_ — this is to ensure that npm isn't accidentally upgraded to a broken version for the new node version.  However, not updating the npm version may itself cause [problems](https://github.com/search?l=&q=%22npm+does+not+support+Node.js%22&type=Issues).
```

Comment:
"Not updating npm" won't ever cause problems, because whatever npm comes with node will continue to work.
```suggestion
 Note that reinstalling packages _explicitly does not update the npm version_ — this is to ensure that npm isn't accidentally upgraded to a broken version for the new node version.
```